src/EventSubscriber/CachePurge/ContentEventsSubscriber.php

```diff
@@ -74,7 +75,7 @@ public function onDeleteVersionEvent(DeleteVersionEvent $event): void
         $contentId = $event->getVersionInfo()->getContentInfo()->id;
 
         $this->purgeClient->purge([
-            'delete-versions-' . $contentId,
+            ContentTagInterface::CONTENT_VERSION_PREFIX . $contentId,
```

Comment:
note, I assume the `delete-versions-` was a typo here and it should have been `content-version-`. If there is a new tag for our delete domain, let me know ;)